ArduPlane/quadplane.cpp

```diff
@@ -772,7 +769,7 @@ bool QuadPlane::setup(void)
     // param count will have changed
     AP_Param::invalidate_count();
 
-    gcs().send_text(MAV_SEVERITY_INFO, "QuadPlane initialised");
+    ::printf("QuadPlane initialised, class: %s, type: %s\n", motors->get_frame_string(), motors->get_type_string());
```

Comment:
This is a third variant of the way we're presenting the class/type.

I'm going to suggest a fourth:
`QUAD/PLUS`

just because.

Reply:
I like that... what does @rmackay9 think?